vendor.conf

```diff
@@ -1,11 +1,11 @@
-github.com/agl/ed25519 d2b94fd789ea21d12fac1a4443dd3a3f79cda72c
+github.com/agl/ed25519 5312a61534124124185d41f09206b9fef1d88403
 github.com/Azure/go-ansiterm d6e3b3328b783f23731bc4d058875b0371ff8109
-github.com/beorn7/perks 4c0e84591b9aa9e6dcfdf3e020114cd81f89d5f9
+github.com/beorn7/perks 3a771d992973f24aa725d07868b467d1ddfceafb
 github.com/containerd/containerd 08f7ee9828af1783dc98cc5cc1739e915697c667
 github.com/containerd/continuity d8fb8589b0e8e85b8c8bbaa8840226d0dfeb7371
-github.com/coreos/etcd v3.2.1
+github.com/coreos/etcd v3.3.8
```

Comment:
3.3.9 was released

Also; I've been punting on this one (SwarmKit is also on a very old version, but I _think_ there were some complications there updating, but should check)

Reply:
oh :thinking: